crates/starknet-server/src/api/http/endpoints.rs

```diff
@@ -0,0 +1,112 @@
+use crate::api::models::transaction::TransactionHashHex;
+
+use super::{
+    error::HttpApiError,
+    models::{
+        AbortedBlocks, AbortingBlocks, Balance, ContractAddress, ContractCode, CreatedBlock,
+        FeeToken, ForkStatus, MessageFromL2, MessageHash, MessageToL2, MintTokens,
+        MintTokensResponse, Path, PostmanLoadL1MessagingContract, PredeployedAccount, Time,
+    },
+    HttpApiHandler, HttpApiResult,
+};
+
+use axum::{extract::Query, Extension, Json};
+
+pub(crate) async fn is_alive() -> HttpApiResult<()> {
+    Ok(())
+}
+
+/// Dumping and loading
+pub(crate) async fn dump(
+    Json(_path): Json<Path>,
+    Extension(_state): Extension<HttpApiHandler>,
+) -> HttpApiResult<()> {
+    Err(HttpApiError::GeneralError)
+}
+
+pub(crate) async fn load(
+    Json(_path): Json<Path>,
+    Extension(_state): Extension<HttpApiHandler>,
+) -> HttpApiResult<()> {
+    Err(HttpApiError::PathNotFound)
+}
+
+/// Postman
+pub(crate) async fn postman_load(
+    Json(_l1_contract): Json<PostmanLoadL1MessagingContract>,
+) -> HttpApiResult<()> {
+    Err(HttpApiError::GeneralError)
+}
+
+pub(crate) async fn postman_flush() -> HttpApiResult<()> {
+    Err(HttpApiError::GeneralError)
+}
+
+pub(crate) async fn postman_send_message_to_l2(
+    Json(_data): Json<MessageToL2>,
+) -> HttpApiResult<Json<TransactionHashHex>> {
+    Err(HttpApiError::GeneralError)
+}
+
+pub(crate) async fn postman_consume_message_from_l2(
+    Json(_data): Json<MessageFromL2>,
+) -> HttpApiResult<Json<MessageHash>> {
+    Err(HttpApiError::GeneralError)
+}
+
+/// Blocks
+pub(crate) async fn create_block() -> HttpApiResult<Json<CreatedBlock>> {
+    Err(HttpApiError::GeneralError)
+}
+
+pub(crate) async fn abort_blocks(
+    Json(_data): Json<AbortingBlocks>,
+) -> HttpApiResult<Json<AbortedBlocks>> {
+    Err(HttpApiError::GeneralError)
+}
+
+/// Restart
+pub(crate) async fn retart() -> HttpApiResult<()> {
+    Err(HttpApiError::GeneralError)
+}
+
+/// Time
+pub(crate) async fn set_time(Json(_data): Json<Time>) -> HttpApiResult<()> {
+    Err(HttpApiError::GeneralError)
+}
+
+pub(crate) async fn increase_time(Json(_data): Json<Time>) -> HttpApiResult<()> {
+    Err(HttpApiError::GeneralError)
+}
+
+/// Accounts
+pub(crate) async fn predeployed_accounts() -> HttpApiResult<Json<Vec<PredeployedAccount>>> {
+    Err(HttpApiError::GeneralError)
+}
+
+pub(crate) async fn get_contract_code(
```

Comment:
Every HTTP method is present in the PR, because it has a curl example in the https://0xspaceshard.github.io/starknet-devnet/docs/intro. I don't know which will not be supported. If this for sure will not be supported I will remove it

Reply:
I meant to say that this method very much looks like a part of feeder gateway API and we don't plan to support feeder gateway API

Reply:
Then I missed some of statements in previous planning and I was not aware that this method was part of feeder gateway API, anyway should I remove it?

Reply:
Btw I'm not sure which curl example you are talking about, there is nothing on the link.
This is the implementation of `/get_code` in the feeder gateway blueprint:
https://github.com/0xSpaceShard/starknet-devnet/blob/7840c9dc1735d9bebb9d7bd0264714a502587c0b/starknet_devnet/blueprints/feeder_gateway.py#L191

I think we should remove this.

Reply:
Removed in [1f103e8](https://github.com/0xSpaceShard/starknet-devnet-rs/pull/8/commits/1f103e8060fd8fd075399c215d574539c42fcb54)